project/frontend_views/views.py

```diff
@@ -286,7 +286,7 @@ def civi2csv(request, thread_id):
     import csv
     thread = thread_id
     response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment, filename=' + thread + '.csv'
+    response['Content-Disposition'] = 'attachment; filename=' + thread + '.csv'
```

Comment:
While small, consider fixing this bug in a separate PR, with related issue. That makes it easier to track and understand why this code was added.

Reply:
I.e. this PR should only focus on Sentry.

Reply:
Its in separate commit so in the history it'll be a different change.
Nevermind, if its easier to manage different PRs that's fine will do it.